docs/user/gcp/customization.md

```diff
@@ -0,0 +1,106 @@
+# GCP Platform Customization
+
+Beyond the [platform-agnostic `install-config.yaml` properties](../customization.md#platform-customization), the installer supports additional, GCP-specific properties.
+
+* `projectID` (required string): The project where the cluster should be created.
+* `region` (required string): The GCP region where the cluster should be created.
+* `network` (optional string): The ID of an existing GCP VPC where the cluster infrastructure should be provisioned.
+* `controlPlaneSubnet` (optional string): The ID of an existing GCP subnet which should be used by the cluster control plane.
+* `computeSubnet` (optional string): The ID of an existing GCP subnet which should be used by the cluster nodes.
+* `defaultMachinePlatform` (optional object): Default [GCP-specific machine pool properties](#machine-pools) which apply to [machine pools](../customization.md#machine-pools) that do not define their own GCP-specific properties.
+
+## Machine pools
+
+* `type` (optional string): The [GCP machine type][machine-type].
+* `zones` (optional array of strings): The availability zones used for machines in the pool.
+
+## Installing to Existing Networks & Subnetworks
+
+The installer can use an existing VPC and subnets when provisioning an OpenShift cluster. If one of `network`, `controlPlaneSubnet`, or `computeSubnet` is specified, all must be specified. The installer will use these existing networks when creating infrastructure such as VM instances, load balancers, firewall rules, and DNS zones.
```

Comment:
with the current validation setup, the requirement is that these should in the same project as the cluster.